### PR TITLE
Add more tests, pragma - no covers

### PR DESCRIPTION
**Pull request by bouwew**

### CODE REVIEW

File: custom_components/plugwise/gateway.py [hunk 1]
```diff
@@ -75,18 +75,21 @@ async def async_setup_entry_gw(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     api.get_all_devices()
 
     # Migrate to the new smile hostname as unique_id
+    # This migration is from several years back, can probably be removed
```
Comment (CoMPaTech) [inline custom_components/plugwise/gateway.py:78]: +1

**Review by CoMPaTech — COMMENTED** [review]
Looking good

**Comment by bouwew** [comment]
image

**Comment by bouwew** [comment]
All remaining are Stick-related.

**Review by CoMPaTech — APPROVED** [review]
See note on removal

**Pull request merged**